site/content/cluster/binaries_cluster.md

```diff
@@ -0,0 +1,276 @@
+---
+linktitle: "Binaries"
+weight: 2
+---
+
+# Creating an M3 Cluster with Binaries
+
+This guide shows you the steps involved in creating an M3 cluster using M3 binaries, typically you would automate this with infrastructure as code tools such as Terraform or [Kubernetes](/docs/operator).
+
+{{% notice note %}}
+This guide assumes you have read the [quickstart](/docs/quickstart), and builds upon the concepts in that guide.
+{{% /notice %}}
+
+## M3 Architecture
+
+Here's a typical M3 deployment:
+
+<!-- TODO: Update image -->
+
+![Typical Deployment](/cluster_architecture.png)
```

Comment:
This image looks pretty blurry in the UI. Can we do anything about this?

Reply:
It's a pretty old picture, I think I'll keep it as is, and update pictures generally later.